src/app/application.cpp

```diff
@@ -403,15 +403,63 @@ void Application::processParams(const QStringList &params)
         return;
     }
 #endif
+    BitTorrent::AddTorrentParams torrentParams;
+    bool skipTorrentDialog = false;
 
     foreach (QString param, params) {
         param = param.trimmed();
+
+        // Process strings indicating options specified by the user. Use of
+        // any of these should prevent the "Add new torrent" dialog from
+        // opening:
+
+        if (param.startsWith("@savePath=")) {
+            torrentParams.savePath = param.mid(10);
+            skipTorrentDialog = true;
```

Comment:
Well being that I have a hard time implementing features that aren't quite finished or don't quite do *The Right Thing* I went ahead and modified `AddNewTorrentDialog` so that it works with my new command line options.